pynicotine/gtkgui/userbrowse.py

```diff
@@ -336,6 +374,44 @@ def __init__(self, userbrowses, user):
                 (">" + _("User"), self.user_popup)
             )
 
+        """ Shortcut Key Bindings (Tool Bar) """
+
+        setup_accelerator("<Primary>f", self.Main, self.on_search_accelerator)  # Find focus
+        setup_accelerator("<Primary>f", self.FolderTreeView, self.on_search_accelerator)
+        setup_accelerator("<Primary>f", self.FileTreeView, self.on_search_accelerator)
+
+        setup_accelerator("<Primary>g", self.Main, self.on_search_next_accelerator)  # Find on_search Enter (repeat)
+        setup_accelerator("<Primary>g", self.FolderTreeView, self.on_search_next_accelerator)
+        setup_accelerator("<Primary>g", self.FileTreeView, self.on_search_next_accelerator)
+        setup_accelerator("F3", self.Main, self.on_search_next_accelerator)
```

Comment:
These function declarations each require entry into 3 objects, which would make a For loop more appropriate (to iterate through a list of: Main; FolderTreeView; FileTreeView) to save on code lines, but I don't know how to implement this.

Reply:
Isn't it enough to add accelerators for Main? They should work in any child widgets, unless there's a bug.

Reply:
Correct, perhaps I was brushing with broad strokes here. Your new implementation seems much more robust so these forceful overrides are not needed so much anymore.

This same token like goes for the on_folder~ and on_file~ functions aswell, in future my plan is to condense these down by structuring those functions which are shared between the two Treeview widgets, but my ability for this level of logic doesn't come easily.